src/index.js

```diff
@@ -40,12 +29,13 @@ class Masq {
     this.sws = {}
     this.hubs = {}
     this.app = app
-    this.channel = null
-    this.challenge = null
+    this.getProfilesChannel = null
```

Comment:
you should be able to just define local variables right (for the 4 variables) ?

Reply:
yes those variables can be defined as local, I had kept it as it was before.
I updated it with local variables now